portal-impl/src/portal.properties

```diff
@@ -6924,6 +6924,21 @@
     #
     feature.flag.LPS-174417.type=beta
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER7__NUMBER9__NUMBER0__NUMBER3__NUMBER5__PERIOD_TITLE
+    #
+    feature.flag.LPS-179035.title=Render Publication Changes with Data Set Display
+
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER7__NUMBER9__NUMBER0__NUMBER3__NUMBER5__PERIOD_DESCRIPTION
+    #
+    feature.flag.LPS-179035.description=Improve performance of "Review changes" screen.
+
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER7__NUMBER9__NUMBER0__NUMBER3__NUMBER5__PERIOD_TYPE
+    #
+    feature.flag.LPS-179035.type=beta
+
```

Comment:
We don't need to define the type yet. It will be added when it passes PM review.